src/deck/backends/deck_backend_onewire.c

```diff
@@ -0,0 +1,121 @@
+/**
+ * ,---------,       ____  _ __
+ * |  ,-^-,  |      / __ )(_) /_______________ _____  ___
+ * | (  O  ) |     / __  / / __/ ___/ ___/ __ `/_  / / _ \
+ * | / ,--´  |    / /_/ / / /_/ /__/ /  / /_/ / / /_/  __/
+ *    +------`   /_____/_/\__/\___/_/   \__,_/ /___/\___/
+ *
+ * Crazyflie control firmware
+ *
+ * Copyright (C) 2025 Bitcraze AB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, in version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * deck_backend_onewire.c - OneWire deck discovery backend
+ */
+
+#include "deck_discovery.h"
+#include "deck.h"
+#include "ow.h"
+
+#define DEBUG_MODULE "DECK_BACKEND_OW"
+#include "debug.h"
+
+// Uncomment to enable debug prints for OneWire backend
+// #define DEBUG_ONEWIRE_BACKEND
+
+#ifdef DEBUG_ONEWIRE_BACKEND
+#define OW_BACKEND_DEBUG(fmt, ...) DEBUG_PRINT(fmt, ## __VA_ARGS__)
+#else
+#define OW_BACKEND_DEBUG(...)
+#endif
+
+// OneWire backend state
+static uint8_t totalDecks = 0;
+static uint8_t currentDeck = 0;
+static DeckInfo deckBuffer; // Buffer for returning deck info
+
+// Dummy driver for decks that do not have a driver implemented
+extern const DeckDriver dummyDriver;
+
+static bool owBackendInit(void) {
+    OW_BACKEND_DEBUG("Initializing OneWire backend\n");
+    currentDeck = 0;
+    totalDecks = 0;
+
+    // Always initialize OneWire (needed for memory system)
+    owInit();
+
+#ifndef CONFIG_DEBUG_DECK_IGNORE_OWS
```

Comment:
Similar comment as for the forced deck, can this just be the OW backend that is not being compiled?